docs/pages/eas-update/debug.mdx

```diff
@@ -1,69 +1,87 @@
+<ImageSpotlight
+  alt="Deployment has no updates"
+  src="/static/images/eas-update/deployments-wrong-channel.png"
+  style={{ maxWidth: 600 }}
+/>
 
-Whenever we run `eas build`, the `npx expo prebuild` command is run on our project on EAS' servers to unpack the **android** and **ios** directories that contain native files. This makes it so EAS Build can build any project, whether it includes the native files or not.
+If the deployment channel is unexpected, it means your build was not built with the correct channel. To fix this, [configure your channel](/#configure-channel).
 
-If our project does not have **android** or **ios** directories, we can make commit any existing changes, then run `npx expo prebuild` to inspect the project state that EAS Build will act on. After running this, look for the following files: **android/app/src/main/AndroidManifest.xml** and **ios/your-project-name/Supporting/Expo.plist**.
+### Unexpected runtime version
 
-In each, we expect to see configuration for the EAS Update URL and the runtime version. Here are the properties we'd expect to see in each file:
+<ImageSpotlight
+  alt="Deployment has no updates"
+  src="/static/images/eas-update/deployments-wrong-runtime.png"
+  style={{ maxWidth: 600 }}
+/>
 
-**AndroidManifest.xml**
+If the deployment runtime version is unexpected, it means your build was not built with the correct runtime version. To fix this, [configure your runtime version](/#configure-runtime-version).
 
-```xml
-...
-<meta-data android:name="expo.modules.updates.EXPO_RUNTIME_VERSION" android:value="your-runtime-version-here"/>
-<meta-data android:name="expo.modules.updates.EXPO_UPDATE_URL" android:value="https://u.expo.dev/your-project-id-here"/>
-...
-```
+### Unexpected branch
 
-**Expo.plist**
+<ImageSpotlight
+  alt="Deployment has no updates"
+  src="/static/images/eas-update/deployments-wrong-branch.png"
+  style={{ maxWidth: 600 }}
+/>
 
-```xml
-...
-<key>EXUpdatesRuntimeVersion</key>
-<string>your-runtime-version-here</string>
-<key>EXUpdatesURL</key>
-<string>https://u.expo.dev/your-project-id-here</string>
-...
-```
+If the deployment has an unexpected branch, you will need to [map your channel to the correct branch](/#map-channel-to-branch).
 
-## EAS Update configuration
+### Missing updates
 
-To debug the state of EAS Update in our project, we'll need to look at multiple spots in the system. Below is a diagram of how EAS Update works and the spots that are useful to inspect when finding the root cause of an issue. In the sections following, we'll inspect and verify these spots and more.
+<ImageSpotlight
+  alt="Deployment has no updates"
+  src="/static/images/eas-update/deployments-no-updates.png"
+  style={{ maxWidth: 600 }}
+/>
 
-<ImageSpotlight alt="Map of debugging spots" src="/static/images/eas-update/debug-map.png" />
+The displayed deployment does not have any updates. To fix this, [publish an update to the branch](/#publish-update).
```

Comment:
Other reason is that the runtime version _of the update_ doesn't match that of the build. This is common enough to note here as well.